crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/ApplicantMapper.java

```diff
@@ -44,9 +48,29 @@ Applicant map(ClientDetails crimeApplyClientDetails) {
         applicant.setHomeAddress(mapAddress(crimeApplyApplicant.getHomeAddress()));
         applicant.setPostalAddress(mapAddress(crimeApplyApplicant.getCorrespondenceAddress()));
 
+        mapEmploymentStatus(applicant ,crimeApplyResponse);
+
         return applicant;
     }
 
+    private void mapEmploymentStatus(Applicant applicant, MaatApplicationExternal crimeApplyResponse) {
+        EmploymentStatus employmentStatus = new EmploymentStatus();
+        if (!crimeApplyResponse.getMeansPassport().isEmpty() &&
```

Comment:
null check on means passport?

Reply:
means passport is a required object, so no need null check.